http/http-advanced/src/test/java/io/quarkus/ts/http/advanced/HttpAdvancedIT.java

```diff
@@ -15,7 +15,8 @@
 public class HttpAdvancedIT extends BaseHttpAdvancedIT {
 
     @KeycloakContainer(command = { "start-dev", "--import-realm", "--hostname-strict=false" })
-    static KeycloakService keycloak = new KeycloakService(DEFAULT_REALM_FILE, DEFAULT_REALM, DEFAULT_REALM_BASE_PATH);
+    static KeycloakService keycloak = new KeycloakService(DEFAULT_REALM_FILE, DEFAULT_REALM, DEFAULT_REALM_BASE_PATH)
```

Comment:
Do we need these for every test with keycloak? If not, what is so special about this one?

Reply:
I don't know much about how Keycloak works, but yes these attributes are necessary in our test context with Keycloak configuration. I tried removing them keeping just the `DEFAULT_REALM_FILE` and the test failed, so `DEFAULT_REALM` and `DEFAULT_REALM_BASE_PATH` are also required for OIDC setup.

Reply:
I was talking about `JAVA_OPTS` part on the next line

Reply:
I've fixed the memory issue in our Jenkins job for this test which was the main issue.
 After analyzing other modules using KeycloakContainer and our KeycloakService, I found dependency differences that likely require less memory. For example, this specific module includes Netty dependencies:
```<dependency>
            <groupId>io.netty</groupId>
            <artifactId>netty-transport-native-epoll</artifactId>
            <classifier>linux-x86_64</classifier>
        </dependency>
        <dependency>
            <groupId>io.netty</groupId>
            <artifactId>netty-transport-native-kqueue</artifactId>
            <classifier>osx-x86_64</classifier>
        </dependency>
 ```

In this issue, with the netty version we use` 4.1.111.Final `: 
        https://github.com/netty/netty/issues/14221 - Memory leak in SSLHandler
      These dependencies may be consuming more memory on our Jenkins job machine...
      Look for example this netty issue:
      https://github.com/netty/netty/issues/8159 - Significant epoll native memory usage jump
      Not sure if related but also talk about epoll and use of much native memory allocated:
      https://github.com/netty/netty/issues/8288

Reply:
But I've just realized that those dependencies are not related to the problematic test `HttpAdvancedIT` they are used for `DomainSocketIT`, I will investigate deeper, thanks @fedinskiy

Reply:
It seems the test that increases memory consumption and leads to the docker Keycloak container fail is: `http2Server`
https://github.com/quarkus-qe/quarkus-test-suite/blob/c9f76aef2d947f4ec64b9bb9a7385ee3121ee558/http/http-advanced/src/test/java/io/quarkus/ts/http/advanced/BaseHttpAdvancedIT.java#L128

Reply:
I've tweaked the http2Server test with my last changes to fix the container issue.

Reply:
Do we still need `JAVA_OPTS` then?

Reply:
No, we don't, I've removed it, thanks.